happi/cli.py

```diff
@@ -1,93 +1,43 @@
 """
 This module defines the ``happi`` command line interface.
 """
-import argparse
+import ast
 import fnmatch
 import json
 import logging
 import os
+# allows arrow key navigation in prompt
```

Comment:
The import isn't used anywhere, it fixes the issue just by being imported.  I'm ok taking out that comment, but thought it'd be confusing otherwise

Reply:
Interesting that it's on-import behavior.

I wish there were some explicit call we could make here to enable the functionality instead. Docs: https://docs.python.org/3/library/readline.html
Nothing relevant we could use/set as far as I can see.

Reply:
https://docs.python.org/3/library/functions.html#input

It's actually mentioned in the `input()` docs.  Doesn't make it any better, but there it is...

Reply:
I was trying to get a feel for what all of this means and found that `python -i` implicitly imports `readline` and therefore handles `input()` differently. Bizarre behavior all-around, definitely correct to import it here...

Reply:
Weird indeed... What about changing _where_ the import happens? Would it be OK if the import were in `happi.cli.main()` (or the name-changed version, I forget)?

While I can't imagine a scenario where another package would complain about readline being imported, it seems reasonable to shove the on-import happenings in the happi CLI handler instead.

Reply:
It is definitely pretty screwed up that someone importing a package somewhere in any module somewhere could change your keybind handlings subtly in specific cases. I think generally people want the readline behavior though, thankfully.

I have no strong opinions on if moving the import matters or not.

Reply:
I might leave it where it is, since it is technically possible to run the commands individually without invoking the parent functions (`happi.cli.main()` or `happi.cli.happi_cli()`).  It requires building the correct context object, but it is something that can happen